metrics/inspector_prioritization.py

```diff
@@ -0,0 +1,274 @@
+import boto3
+import pandas as pd
+import numpy as np
+from arcgis.gis import GIS
+from arcgis.features import FeatureLayer
+from arcgis.geometry.filters import intersects
+from sodapy import Socrata
+
+import os
+import logging
+
+from config import (
+    SEGMENTS_FEATURE_SERVICE_URL,
+    DAPCZ_FEATURE_SERVICE_URL,
+    ROW_INSPECTOR_SERVICE_URL,
+)
+from utils import get_logger, s3_csv_to_df, df_to_socrata_dataset
+
+# AWS Credentials
+AWS_ACCESS_ID = os.getenv("EXEC_DASH_ACCESS_ID")
+AWS_PASS = os.getenv("EXEC_DASH_PASS")
+BUCKET = os.getenv("BUCKET_NAME")
+
+# Socrata Credentials
+SO_WEB = os.getenv("SO_WEB")
+SO_TOKEN = os.getenv("SO_TOKEN")
+SO_KEY = os.getenv("SO_KEY")
+SO_SECRET = os.getenv("SO_SECRET")
+DATASET = os.getenv("PRIORITY_DATASET")
+
+# AGOL Credentials
+AGOL_USERNAME = os.getenv("AGOL_USERNAME")
+AGOL_PASSWORD = os.getenv("AGOL_PASSWORD")
+
+PERMITS_FILE = "row_inspector_permit_list.csv"
+SEGMENTS_FILE = "row_inspector_segment_list.csv"
+
+
+def number_of_segments_scoring(permits, segments):
+    """
+    10 points for permits with more than 1 segment, 5 points otherwise.
+
+    Returns
+    -------
+    permits - with a count_segment_scoring column with the scores based on the number of segments
+
+    """
+
+    counts = segments.groupby("FOLDERRSN")["PROPERTYRSN"].count()
+    counts = counts.rename("count_segments")
+    permits = permits.merge(counts, left_on="FOLDERRSN", right_index=True, how="left")
+    permits["count_segments"] = permits["count_segments"].fillna(0)
+
+    # 10 points for permits with more than 1 segment, 5 points otherwise:
+    permits["count_segment_scoring"] = np.where(permits["count_segments"] > 1, 10, 5)
```

Comment:
i don't know if this is really practical to implement, but it could be nice to collect all these magic numbers used for scoring into named vars at the top of the script, or a dict. that would make life easier the next time we need to tweak the scoring parameters.